lib/rspec/matchers/built_in/output.rb

```diff
@@ -136,13 +146,25 @@ def self.capture(_block)
       end
 
       # @private
-      module CaptureStdout
-        def self.name
+      class CapturedStream < StringIO
+        attr_accessor :as_tty
+
+        def tty?
+          as_tty || super
```

Comment:
Forcing this to false should also be possible.

```suggestion
          return super if as_tty.nil?
          as_tty
```

Reply:
I was suggestting this to leave a `as_not_tty` or `as_tty(false)` future [or could even be added here] to be explicit about what this setting is as `StringIO` is not under our control and could be monkey patched.

Reply:
Makes sense, thank you!